scripts/16_create-codebook.R

```diff
@@ -7,6 +8,9 @@ library(stringr)
 
 trials <- read_rds(here("data", "processed", "trials.rds"))
 
+
+# 1. Prepare codebook structure -------------------------------------------
+
 # Get intovalue codebook for descriptions
 iv_codebook <-
 
```

Comment:
General comment: I'm wondering if we even need to read in the original codebook since we anyway then replace all the descriptions with the most up-to-date ones. An option could be to start directly with building `codebook-structure`, and then join in the info from the google sheet. We could still refer to the original codebook in a comment. What do you think?

Reply:
👋 @delwen ! Thanks a lot for this feedback and for taking the time to talk it through. I refactored a bit and added more comments and documentation, and I hope this brings more clarity to the convolution. If this addresses your concern, you should now be able to merge the PR. If not, please let me know.

Reply:
oops! good catch, forgot to stage that for the last commit. Added in 1578b060e0f30de614373283e20c6cbf678a0878.